tests/security/openssl/tlsv1_3.pm

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: openssl 1.1.1 supports and negotiates by default the new TLS 1.3 protocol.
+#          Applications that leave everything to the openssl library will automatically
+#          start to negotiate the TLS 1.3 protocol. However, many packages have their
+#          own settings which override the library defaults and these either have to be
+#          recompiled against openssl 1.1.1 or might even need extra patching.
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#64992, tc#1744100
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+use apachetest;
+
+sub run {
+    select_console 'root-console';
+    setup_apache2(mode => 'SSL');
+
+    # List the supported ciphers and make sure TLSV1.3 is there
+    validate_script_output 'openssl ciphers -v', sub { m/TLSv1\.3.*/xg };
+
+    # Establish a transparent connection to apache server to check the TLS protocol
+    validate_script_output 'echo | openssl s_client -connect localhost:443 2>&1', sub { m/TLSv1\.3.*/xg };
+
+    #  Transfer a URL to check the TLS protocol
```

Comment:
Please delete the extra space in line 40.
Btw, why not add this test case into fips test case since openssl is part of fips' testings?

Reply:
Thanks, Will do!

I agree with you that fips can cover this test as well, but for openssl itself, we need cover it in non-fips mode as well (or we can test the same on ppcle64 for now, right?)

Reply:
Done

Reply:
I also have the curiosity about it as Lily mentioned, we have the folder are located ../tests/fips/openssl/, but you create a new folder as ../security/openssl/ .  Does Lily mention the same problem as mine? :)

Reply:
> I also have the curiosity about it as Lily mentioned, we have the folder are located ../tests/fips/openssl/, but you create a new folder as ../security/openssl/ . Does Lily mention the same problem as mine? :)

Almost the same. But I am not sure if it is a good idea to put the code of this test case to fips dir and make openQA run it in fips openssl also.
We need run it without "FIPS_ENABLED" for our feature only (but this feature is also a preparation for openssl fips), so we also need to set "FIPS_ENABLED" for openssl TSL.

Reply:
> Thanks, Will do!
> 
> I agree with you that fips can cover this test as well, but for openssl itself, we need cover it in non-fips mode as well (or we can test the same on ppcle64 for now, right?)

For non-fips test cases we can unset "FIPS_ENABLED" and run them on all arches.
Let's discuss it in tomorrow's scrum meeting :)

Reply:
As we discussed, mv the code to fips/openssl